### ISSUE TITLE
Option to ignore age of crops and show as being placed correctly

### ISSUE THREAD

**Issue by shruggiexo** [maruohon/litematica#721]
**Is your feature request related to a problem? Please describe.**
(In the Schematic Verifier)
It is annoying when using this to build a farm, and have the age of crops such as cacti cause the blocks to be marked as "Wrong State". This block state isn't necessary to check for. When ignoring those issues, they then won't show as being correct with the the "Correct State" filter selected. This carries over to the placement rendering where the blocks are highlighted with being in the wrong state.

**Describe the solution you'd like**
Would be ideal to have an "Ingore crop age" option in the config, without needing to modify the schematic itself. This would mean crops wouldn't be highlighted as the incorrect state, and in the verifier, would show as being correct

**Describe alternatives you've considered**
N/A

**Additional context**
![image](https://github.com/maruohon/litematica/assets/57070409/0d08aad6-1a9c-4af5-93d0-7408d7e970d4)

**Comment by DereC4** [comment]
I second this, it makes building crop farms hard without using worldedit to first set all crops to age 0

**Comment by Enethion** [comment]
I'm bumping this up. It's been 3 years, and nothing changed on the matter?

**Comment by maruohon** [comment]
@Enethion Exactly. That's usually what happens when you get burnt out on something and don't work on it anymore.

**Comment by Enethion** [comment]
@maruohon fair 🤔

**Comment by bocchi-the-cat** [comment]
this litematica is abandoned (should be archived ffs, idk why it isnt)

you can find the maintaned litematica [here](https://github.com/sakura-ryoko/litematica)

you should see it in the newest 26.2 version (on modirinth too)  as a setting "ignoreCropAge"


btw the one from modirith is also sakura's
click on view source on modirith

**Comment by Enethion** [comment]
> you can find the maintaned litematica [here](https://github.com/sakura-ryoko/litematica)
> 
> you should see it in the newest 26.2 version (on modirinth too) as a setting "ignoreCropAge"

I have. If you check Sakura's github for litematica, I added this issue and it is marked as completed.

Thank you anyway.

**Comment by bocchi-the-cat** [comment]
this is for all the other people that find this (it only support 26.2+ afaik) + if someone wants to close it, since this is the first repo that shows up when searching "litematica"
i could add it to this repo so the issue gets closed but i dont understead the branch structure :(

in fact im the person that added that to litematica!

https://github.com/sakura-ryoko/litematica/commit/785394e0ee6fd5c58cc7fc45eba099c9020b8913

<img width="531" height="82" alt="Image" src="https://github.com/user-attachments/assets/d75cc762-e8ca-4fd2-ae70-ca62d14caa68" />

**Comment by DereC4** [comment]
Would really appreciate a pin then to redirect issues to the forked repo 

<img width="1171" height="302" alt="Image" src="https://github.com/user-attachments/assets/399eef02-c608-4f45-a8d9-495efdb64931" />

And then we could clean up existing issues if finished
